### PR TITLE
fix eastwest gen

### PR DESCRIPTION
**Pull request by stevenctl**
- better std err dump
- probably due to using `istioctl` in script, can we rely on `istioctl` being the built one, or should we give `istio.io/istio/out/...` path? 

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pkg/test/framework/components/istio/eastwest.go [hunk 1]
```diff
@@ -58,15 +59,22 @@ func (i *operatorComponent) deployEastWestGateway(cluster resource.Cluster) erro
 	cmd := exec.Command(genGatewayScript, generateSettings...)
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env,
+		"ISTIOCTL="+filepath.Join(env.LocalOut, "istioctl"),
```
Comment (stevenctl) [inline pkg/test/framework/components/istio/eastwest.go:62]: A user should probably be setting PATH to select an istioctl version, but this is for tf compat.

**Review by howardjohn — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Comment by stevenctl** [comment]
@howardjohn @ericvn you may want to take another look – the first bit was just to get some debug info. Is there a reason we don't use ISTIO_OUT for the postsubmit target?

**Review by howardjohn — APPROVED** [review]

**Pull request merged**